src/app/services/handlers/security-handler.service.ts

```diff
@@ -236,7 +237,8 @@ export class SecurityHandlerService {
       );
   }
 
-  dataModelAccess(element: Securable & Finalisable) {
+
+  elementAccess(element) : Access {
```

Comment:
Is it possible to add a type to `element`? `Securable` gives you the `availableActions` property. For this bit:

```ts
showNewVersion: element.finalised
```

`element` should be `Finalisable`. So maybe try a union?

```ts
elementAccess(element: Securable | Securable & Finalisable): Access;
```

But I'm just musing out loud, don't know if that works.

Reply:
Also, do we want to link this PR to issue #143 as well? The PR would then close both at once.